src/style.css

```diff
@@ -1,3 +1,74 @@
+* {
+  margin: 0;
+  padding: 0;
+  border: none;
+  list-style: none;
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
 }
+
+header {
+  font-family: Roboto, sans-serif;
+  color: #0000;
```

Comment:
The color value '#0000' is not valid. If you meant to use transparent black, use 'rgba(0, 0, 0, 0)' or 'transparent' instead.